projects/index.html

```diff
@@ -0,0 +1,92 @@
+---
+title: Projects
+layout: default
+banner: In 2021, Summer of Bitcoin is funding 51 student projects for 20 open source organizations working on bitcoin and related technologies.
+---
+
+<!-- Navigation-->
+        <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
+            <div class="container">
+                <a class="navbar-brand" href="#page-top">Summer of Bitcoin</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                    Menu
+                    <i class="fas fa-bars ms-1"></i>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarResponsive">
+                    <ul class="navbar-nav text-uppercase ms-auto py-4 py-lg-0">
+                        <li class="nav-item"><a class="nav-link" href="../index.html#about">About</a></li>
```

Comment:
Pretty sure these should all just point back to the main page without specifying the index.html part
```suggestion
                        <li class="nav-item"><a class="nav-link" href="/#about">About</a></li>
```